Framework/CurveFitting/src/Algorithms/SplineSmoothing.cpp

```diff
@@ -303,10 +324,10 @@ void SplineSmoothing::addSmoothingPoints(const std::set<int> &points,
  * @param row :: The row of spectra to use
  */
 void SplineSmoothing::selectSmoothingPoints(
-    MatrixWorkspace_const_sptr inputWorkspace, size_t row) {
+    MatrixWorkspace_const_sptr inputWorkspace, const size_t row) {
```

Comment:
Ahhh - I was having a forgetful moment going the long way around and doing `myWS->get()` to remove the wrapping without realising the `*` operator did the same. 
I'll change that as its a lot cleaner passing a pointer directly.